jms/src/main/scala/akka/stream/alpakka/jms/Exceptions.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.jms
+
+/**
+ * Marker trait indicating that the exception thrown is intermittent. The failed operation might succeed if tried again.
+ */
+trait RetriableException extends Exception
```

Comment:
It makes sense to mark some exception cases as `NonRetriableException`. However the other way around `RetriableException` does not make as much sense.

The essence is that you want to point out for some specific exceptions that a retry makes no sense. However for all other exceptions, a retry might make sense (so there is no real need for the marker trait `RetriableException` especially since the code does not even use this).